erizo/src/erizo/media/ExternalOutput.cpp

```diff
@@ -315,7 +363,11 @@ int ExternalOutput::deliverAudioData_(char* buf, int len) {
 int ExternalOutput::deliverVideoData_(char* buf, int len) {
     if (videoSourceSsrc_ == 0){
       RtpHeader* h = reinterpret_cast<RtpHeader*>(buf);
-      videoSourceSsrc_ = h->getSSRC();
+      if (h->getPayloadType() == RED_90000_PT) {
```

Comment:
I'd turn off FEC in your SDP for any number of reasons.  I've completely abandoned trying to use forward error correct, which removes the RED headers entirely.